rmgpy/reduction/reduction.py

```diff
@@ -268,13 +268,20 @@ def searchTargetIndex(targetLabel, reactionModel):
     """
     Searches for the Species object in the core species
     of the reaction that has the same label as the parameter string.
-    """
 
+    This method does not differentiate between multiple species
```

Comment:
Great idea. I wrote a unittest for the issue. Should I leave the failing test or comment with @work_in_progress?